README.md

```diff
@@ -1,22 +1,74 @@
 # todo-text-stuff
 Stuff I've written for my todo.txt setup.
 
-Currently contains:
+#### Currently contains:
 
-ice_recur : A recurring item generator for todo.txt.  Yes, there are like 14 of these, but I couldn't find a single one that could do "every 2 weeks", so I wrote my own.   REQUIREMENTS: Ruby ; gems: ice_cube, optimist
-  - Run "todo.sh ice_recur --help" for info on how to use this.
-templated_checklists : A file to keep available copies of one-time checklists (i.e. "stuff to take when going swimming")
-  - Read the comments at the top for info on how to use this.
+`ice_recur`: A recurring item generator for todo.txt.  Yes, there are like 14 of these, but I couldn't find a single one that could do "every 2 weeks" and would run the script for previous days that the script was missed if your computer wasn't turned on, so I wrote my own.
+  - REQUIREMENTS: Ruby and gems: ice_cube, optimist, fileutils
+  - Run `todo.sh ice_recur --help` for info on how to use this. Also, command line options posted at the bottom of the README.
 
-Tests:
+`templated_checklists`: A file to keep available copies of one-time checklists (i.e. "stuff to take when going swimming")
+  - Read the comments at the top of the `templated_checklists` file for info on how to use this.
 
-Run ice_recur tests like this if you want pretty output and failure if you miss tests: prove -v --exec "./ice_recur --test" ice_recur
+#### Installation:
 
-Changelog:
+This script goes in `$TODO_DIR/actions/`
 
-ice_recur 1.2   29 Nov 2018: Added support for day_of_week, day_of_year and month_of_year
-ice_recur 1.3   29 Nov 2018: Now uses the timestamp on the .ice_recur_completed file, and runs the schedule for each day since then, so you can run it less than once a day and it'll still work properly.
-ice_recur 1.4   30 Nov 2018: No longer uses todotxt-rb; this means better handling of key/value tags, at least potentially
-ice_recur 1.5    2 Dec 2018: Added -n, -p, and -o.  Won't run twice on the same day anymore.  Additional clarity on what's getting added.
-ice_recur 1.5.1  3 Dec 2018: Bugfix: Start the run the day *after* the last touch to the completed file.
-ice_recur 1.5.2  4 Dec 2018: Added -f
+You put your entries in `$TODO_DIR/ice_recur.txt` , and add something like this:
+
+      ~/bin/todo.sh ice_recur
+
+to your crontab, to run once a day.
+
+Every entry that matches the current day or previous days after `.ice_recur_completed` was last touched will be added, as long as there is no other entry with the same text content.
+
+Comments can be added to the file using `#` but `ice_recur.txt` can contain no blank lines between entries or the script won't execute properly.
```

Comment:
@rlpowell This is true on my computer, so I am not sure if it was meant to be like this.

Reply:
Mine definitely has blank lines and they're no trouble.  Can you tell me exactly what happens to you when you have blank lines?

Reply:
Sorry, this was a mistake. I know I had this problem in an old version but didn't test in the current version. It appears to be fixed.